### PR TITLE
Gh 1920 schedule jobs

### PR DESCRIPTION
**Pull request by m55624**

### CODE REVIEW

**Comment by m55624** [comment]
Closing for fixes.

**Comment by m55624** [comment]
Unreliable tests need fixing.

File: core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java [hunk 1]
```diff
@@ -401,15 +466,21 @@ protected JobDetail executeJob(final OperationChain<?> operationChain, final Con
...
             }
-        };
+        });
 
-        executorService.execute(runnable);
+        return jobDetail;
+    }
 
-        return initialJobDetail;
+    public void runAsync(final Runnable runnable) throws StoreException {
```
Comment (p013570) [inline core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java]: Changing this public interface to add a CheckedException isn't backwards compatible.

**Comment by p013570** [comment]
@m55624 - please can you make sure that jobs that are scheduled via REST are rescheduled
if the REST server is restarted?

File: core/operation/src/main/java/uk/gov/gchq/gaffer/jobtracker/Job.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2018 Crown Copyright
```
Comment (d47853) [inline core/operation/src/main/java/uk/gov/gchq/gaffer/jobtracker/Job.java]:
```suggestion
 * Copyright 2019 Crown Copyright
```

**Pull request merged**